library/waku_thread/config.nim

```diff
@@ -149,6 +165,29 @@ proc parseTopics(jsonNode: JsonNode, conf: var WakuNodeConf) =
   else:
     conf.topics = @["/waku/2/default-waku/proto"]
 
+proc parseRLNRelay(jsonNode: JsonNode, conf: var WakuNodeConf, errorResp: var string): bool =
+  if not jsonNode.contains("rlnRelay"):
```

Comment:
I wonder if we should use the same style as the `wakunode` has. Hyphen-separated terms. 
```suggestion
  if not jsonNode.contains("rln-relay"):
```